tests/idris2/interactive/interactive048/Issue3487.idr

```diff
@@ -0,0 +1,18 @@
+one : {a, b : Nat} -> () -> ()
+one {a, b} c = ?rhs
+
+two : {a : Nat} ->  {b : (Nat, Nat)} -> () -> ()
+two {a=x, b = (c, d)} e = ?rhs2
+
+three : {a : Maybe Nat} -> {b : Nat} -> () -> ()
+three {a = Nothing, b} c = ?rhs3
+three {a = Just x} c = ?rhs4
+
+four : { a : List Nat} -> () -> ()
+four {a = [x,y]} c = ?rhs5
+four {a = xs} c = ?rhs6
+
+five : Nat -> Nat
+five x = (case x of
+  y => ?rhs7 ) -- something y
+
```

Comment:
Can we have test cases of the form `f {- this is f -} x = ?`?

Reply:
I left out that case, but I'll take care of it and add a test.